### PR TITLE
Update @next/font package warning

### PR DESCRIPTION
**Pull request by hanneslund**
Include the command to run the codemod in the warning.

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have a helpful link attached, see [`contributing.md`](https://github.com/vercel/next.js/blob/canary/contributing.md)

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] [e2e](https://github.com/vercel/next.js/blob/canary/contributing/core/testing.md#writing-tests-for-nextjs) tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have a helpful link attached, see [`contributing.md`](https://github.com/vercel/next.js/blob/canary/contributing.md)

## Documentation / Examples

- [ ] Make sure the linting passes by running `pnpm build && pnpm lint`
- [ ] The "examples guidelines" are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md)

### CODE REVIEW

**Review by feedthejim — DISMISSED** [review]

File: packages/next/src/cli/next-dev.ts [hunk 1]
```diff
@@ -200,7 +200,7 @@ const nextDev: CliCommand = async (argv) => {
       Log.warn(
         'Your project has `@next/font` installed as a dependency, please use the built-in `next/font` instead. ' +
           'The `@next/font` package will be removed in Next.js 14. ' +
-          'You can migrate by running the `built-in-next-font` codemod. Read more: https://nextjs.org/docs/messages/built-in-next-font'
+          'You can migrate by running `npx @next/codemod built-in-next-font`. Read more: https://nextjs.org/docs/messages/built-in-next-font'
```
Comment (styfle) [inline packages/next/src/cli/next-dev.ts:203]:
```suggestion
          'You can migrate by running `npx @next/codemod built-in-next-font .`. Read more: https://nextjs.org/docs/messages/built-in-next-font'
```

I think you need to specify the directory with a `.` right?

Comment (hanneslund) [reply packages/next/src/cli/next-dev.ts:203]: You get a prompt to specify the path, but might as well add it 👍

**Review by styfle — APPROVED** [review]

File: packages/next/src/cli/next-dev.ts [hunk 2]
```diff
@@ -200,7 +200,7 @@ const nextDev: CliCommand = async (argv) => {
       Log.warn(
         'Your project has `@next/font` installed as a dependency, please use the built-in `next/font` instead. ' +
           'The `@next/font` package will be removed in Next.js 14. ' +
-          'You can migrate by running the `built-in-next-font` codemod. Read more: https://nextjs.org/docs/messages/built-in-next-font'
+          'You can migrate by running `npx @next/codemod built-in-next-font .`. Read more: https://nextjs.org/docs/messages/built-in-next-font'
```
Comment (jaredpalmer) [inline packages/next/src/cli/next-dev.ts:203]: do we know what package manager they are using? could this be made more relevant?

Comment (hanneslund) [reply packages/next/src/cli/next-dev.ts:203]: Sounds good to me! Opened a PR https://github.com/vercel/next.js/pull/46359

**Pull request merged**